diff --git a/node_modules/@pixi/react/package.json b/node_modules/@pixi/react/package.json
index 32731e9..a959146 100644
--- a/node_modules/@pixi/react/package.json
+++ b/node_modules/@pixi/react/package.json
@@ -37,6 +37,9 @@
       "types": "./dist/index.d.ts",
       "import": "./dist/index.mjs",
       "require": "./dist/index.js"
+    },
+    "react-server": {
+      "import": "./dist/index.mjs"
     }
   },
   "files": [
diff --git a/node_modules/@pixi/react/dist/index.mjs b/node_modules/@pixi/react/dist/index.mjs
index c540026..b02e086 100644
--- a/node_modules/@pixi/react/dist/index.mjs
+++ b/node_modules/@pixi/react/dist/index.mjs
@@ -1,4 +1,4 @@
-import {useReducer, useMemo, createContext, useContext, useEffect, useRef, forwardRef, memo, createElement, useState, useCallback} from "react";
+import {useReducer, useMemo, createContext, useContext, useEffect, useRef, forwardRef, memo, createElement, useState, useCallback} from "react";
 import {DisplayObject, Container, Application as Application$1, Graphics as Graphics$1, Sprite, Text as Text$1,NineSlicePlane, TilingSprite as TilingSprite$1, SimpleRope as SimpleRope$1, SimpleMesh as SimpleMesh$1} from "pixi.js";
 import {render, unmount, applyProps, ReactPixiFiberAsPrimaryRenderer} from "./ReactPixiFiber.js";
 import {BitmapText as BitmapText$1} from "pixi.js";
